CHANGES.md

```diff
@@ -139,6 +139,9 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 * HTTP endpoint targets of DC/OS Diagnostics can be marked optional (DCOS_OSS-5031)
 
+* The configuration parameters `aws_secret_access_key` and `exhibitor_azure_account_key` for exhibitor are now marked as secret and will thus not be revealed in `user.config.yaml` on cluster nodes but will from now on appear only in `user.config.full.yaml` which has stricter read permissions and is not included in DC/OS Diagnostics bundles. (DCOS-51751)
```

Comment:
Is there a reason we're not including the changelog entry on master branch? I'm probably not familiar with the process of updates to changelog.

Reply:
My suggestion is that 1.13 is now GA and no customer has/should have it installed, yet. Thus, from 1.13 to 1.14 there is no change. Make sense? I'm also not too firm with the process.

Reply:
Ah right, okay that makes sense 👍